### PR TITLE
docs: add a section on simple scripts

### PR DESCRIPTION
**Pull request by sourabhxyz**

### CODE REVIEW

File: src/pages/additional-features/simple-scripts.mdx [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Simple Scripts
+
+Atlas supports simple scripts, aka native and timelock scripts. The functionality of simple scripts is explained in ledger design document of Shelley & Mary era [here](https://github.com/IntersectMBO/cardano-ledger) and also in [cardano-node-wiki](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/reference/simple-scripts.md).
```
Comment (4TT1L4) [inline src/pages/additional-features/simple-scripts.mdx:3]:
```suggestion
Atlas supports simple scripts, also known as native and timelock scripts. The functionality of simple scripts is explained in the ledger design documents from the Shelley and Mary eras, which you can find [here](https://github.com/IntersectMBO/cardano-ledger) and in the [cardano-node wiki](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/reference/simple-scripts.md).
```

File: src/pages/additional-features/simple-scripts.mdx [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+
+Simple scripts can be specified via exposed constructors of [`GYSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#t:GYSimpleScript). They can also be read from their JSON representation as `GYSimpleScript` has a [`FromJSON`](https://hackage.haskell.org/package/aeson-2.2.3.0/docs/Data-Aeson.html#t:FromJSON) instance, we also provide a utility function, [`readSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#v:readSimpleScript), to read JSON representation from a file.
```
Comment (4TT1L4) [inline src/pages/additional-features/simple-scripts.mdx:5]:
```suggestion
Simple scripts can be specified using the exposed constructors of [`GYSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#t:GYSimpleScript). They can also be loaded from their JSON representation, as `GYSimpleScript` implements a [`FromJSON`](https://hackage.haskell.org/package/aeson-2.2.3.0/docs/Data-Aeson.html#t:FromJSON) instance. Additionally, we provide the utility function [`readSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#v:readSimpleScript), which reads the JSON representation from a file.
```

File: src/pages/additional-features/simple-scripts.mdx [hunk 3]
```diff
@@ -0,0 +1,7 @@
...
+
+Simple scripts can also be attached to UTxO outputs to be used as reference script inputs later when building transaction. We have an example [here](https://github.com/geniusyield/atlas/blob/main/tests-privnet/GeniusYield/Test/Privnet/SimpleScripts.hs), demonstrating interaction with simple scripts in both scenarios, when being used as reference and otherwise.
```
Comment (4TT1L4) [inline src/pages/additional-features/simple-scripts.mdx:7]:
```suggestion
Simple scripts can also be attached to UTxO outputs, allowing them to be used later as reference scripts when building transactions. You can find an example [here](https://github.com/geniusyield/atlas/blob/main/tests-privnet/GeniusYield/Test/Privnet/SimpleScripts.hs), that demonstrates how to interact with simple scripts, both as reference scripts and in other scenarios.
```

**Review by 4TT1L4 — CHANGES_REQUESTED** [review]
Nice work. Please see my suggestions.

File: src/pages/additional-features/simple-scripts.mdx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+# Simple Scripts
```
Comment (4TT1L4) [inline src/pages/additional-features/simple-scripts.mdx:1]:
```suggestion
# Simple Scripts in Atlas: Native and Timelock Scripts in Cardano
```

File: src/pages/additional-features/simple-scripts.mdx [hunk 5]
```diff
@@ -0,0 +1,7 @@
...
+
+Atlas supports simple scripts, also known as native and timelock scripts. The functionality of simple scripts is explained in the ledger design document of Shelley & Mary era [here](https://github.com/IntersectMBO/cardano-ledger) and also in [cardano-node-wiki](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/reference/simple-scripts.md).
```
Comment (4TT1L4) [inline src/pages/additional-features/simple-scripts.mdx:3]:
```suggestion
Atlas provides support for simple scripts, also referred to as native and timelock scripts, which are key components in building transactions on the Cardano blockchain. These scripts are critical for establishing spending conditions, such as multisig requirements or timelocks. The functionality of simple scripts is detailed in the ledger design document for the Shelley and Mary eras, which can be found [in the Cardano Ledger repository](https://github.com/IntersectMBO/cardano-ledger). Additional documentation can be accessed in the [Cardano Node Wiki's reference section on simple scripts](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/reference/simple-scripts.md).
```

File: src/pages/additional-features/simple-scripts.mdx (hunk 4 above)
Comment (sourabhxyz) [reply src/pages/additional-features/simple-scripts.mdx:1]: I'd just prefer "Simple Scripts", why ".. in Atlas"? And Native/Timelock are just other names of simple scripts.

File: src/pages/additional-features/simple-scripts.mdx [hunk 6]
```diff
@@ -0,0 +1,7 @@
...
+
+Simple scripts can be specified via exposed constructors of [`GYSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#t:GYSimpleScript). They can also be read from their JSON representation as `GYSimpleScript` has a [`FromJSON`](https://hackage.haskell.org/package/aeson-2.2.3.0/docs/Data-Aeson.html#t:FromJSON) instance. We also provide a utility function, [`readSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#v:readSimpleScript), to read JSON representation from a file.
+
+Simple scripts can also be attached to UTxO outputs to be used as reference script inputs later when building transactions. We have an example [here](https://github.com/geniusyield/atlas/blob/main/tests-privnet/GeniusYield/Test/Privnet/SimpleScripts.hs), demonstrating interaction with simple scripts in both scenarios, when being used as reference and otherwise.
```
Comment (4TT1L4) [inline src/pages/additional-features/simple-scripts.mdx:7]:
```suggestion
Simple scripts can also be attached to UTxO outputs, enabling them to be referenced later when building transactions. This allows for reusability of the scripts without embedding them directly into every transaction. We provide an example [here](https://github.com/geniusyield/atlas/blob/main/tests-privnet/GeniusYield/Test/Privnet/SimpleScripts.hs), demonstrating the interaction with simple scripts in two scenarios: when used as reference scripts and when they are embedded directly.
```

File: src/pages/additional-features/simple-scripts.mdx [hunk 7]
```diff
@@ -0,0 +1,7 @@
+# Simple Scripts
+
+Atlas supports simple scripts, also known as native and timelock scripts. The functionality of simple scripts is explained in the ledger design document of Shelley & Mary era [here](https://github.com/IntersectMBO/cardano-ledger) and also in [cardano-node-wiki](https://github.com/input-output-hk/cardano-node-wiki/blob/main/docs/reference/simple-scripts.md).
+
+Simple scripts can be specified via exposed constructors of [`GYSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#t:GYSimpleScript). They can also be read from their JSON representation as `GYSimpleScript` has a [`FromJSON`](https://hackage.haskell.org/package/aeson-2.2.3.0/docs/Data-Aeson.html#t:FromJSON) instance. We also provide a utility function, [`readSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#v:readSimpleScript), to read JSON representation from a file.
```
Comment (4TT1L4) [inline src/pages/additional-features/simple-scripts.mdx:5]:
```suggestion
Simple scripts can be specified via exposed constructors of the `GYSimpleScript` data type, which represents native and timelock scripts in Atlas. This data type can be used for both creating and interacting with simple scripts programmatically. 

You can specify simple scripts in their JSON representation, which is useful for reading and writing scripts from external files. The `GYSimpleScript` data type implements the [`FromJSON`](https://hackage.haskell.org/package/aeson-2.2.3.0/docs/Data-Aeson.html#t:FromJSON) typeclass, allowing for easy conversion from JSON. Additionally, we provide the [`readSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#v:readSimpleScript) utility function to load a script from a JSON file, simplifying the process of working with pre-defined scripts.
```

Comment (sourabhxyz) [reply src/pages/additional-features/simple-scripts.mdx:5]: This suggestion, "...type, which represents native and timelock scripts in Atlas. This data type can be used for both creating and interacting with simple scripts programmatically. " is unnecessary.

File: src/pages/additional-features/simple-scripts.mdx (hunk 4 above)
Comment (4TT1L4) [inline src/pages/additional-features/simple-scripts.mdx:1]:
```suggestion
# Simple Scripts in Atlas: Native and Timelock Scripts in Cardano
```

File: src/pages/additional-features/simple-scripts.mdx (hunk 6 above)
Comment (4TT1L4) [inline src/pages/additional-features/simple-scripts.mdx:7]:
```suggestion
Additionally, we provide the [`readSimpleScript`](https://haddock.atlas-app.io/GeniusYield-Types-Script-SimpleScript.html#v:readSimpleScript) utility function to load a script from a JSON file, simplifying the process of working with pre-defined scripts.
```

**Review by 4TT1L4 — APPROVED** [review]
Thanks for considering my suggestions.

**Pull request merged**